docs/HowTo/Interact/APIs/Engine-API.md

```diff
@@ -0,0 +1,193 @@
+[`--engine-rpc-http-port`](../../../Reference/CLI/CLI-Syntax.md#engine-rpc-http-port) or
+[`--engine-rpc-ws-port`](../../../Reference/CLI/CLI-Syntax.md#engine-rpc-ws-port) options.
+
+The default ports are:
+
+* 8550 over HTTP
+* 8551 over WebSockets
+
+### Host allowlist
+
+To prevent DNS rebinding attacks, Besu checks incoming HTTP request host headers, WebSockets connections, and GraphQL
+requests.
+Besu accepts requests only when hostnames specified using the
+[`--engine-host-allowlist`](../../../Reference/CLI/CLI-Syntax.md#engine-host-allowlist) option matches the request host headers.
+By default, Besu accepts requests and connections from `localhost` and `127.0.0.1`.
+
+!!! important
+
+    This isn't a permissioning feature.
+    If you want to restrict access to the Engine API, we recommend using [authentication](#authentication).
+
+If your application publishes RPC ports, specify the hostnames when starting Besu.
+
+Specify "*" for `--engine-host-allowlist` to effectively disable host protection.
+
+!!! caution
+
+    Specifying "*" for `--engine-host-allowlist` is not recommended for production code.
+
+## Authentication
+
+By default, [Authentication](Authentication.md) for the Engine API is disabled.
+To enable, set the [`--engine-jwt-enabled`](../../../Reference/CLI/CLI-Syntax.md#engine-jwt-enabled) to `true`.
+
+Set the [JWT secret](Authentication.md#jwt-public-key-authentication) by using the [`--engine-jwt-secret`](../../../Reference/CLI/CLI-Syntax.md#engine-jwt-secret) option.
+
+## Sending a payload using the Engine API
+
+### 1. Prepare a payload
+
+Prepare to send a payload by using [`engine_forkchoiceUpdatedV1`](../../../Reference/Engine-API-Methods.md#engine_forkchoiceupdatedv1).
+
+!!! example
+
+    === "curl HTTP request"
+
+        ```bash
+        curl -X POST --data '{"jsonrpc":"2.0","method":"engine_forkchoiceUpdatedV1","params":[{"headBlockHash": "0x3b8fb240d288781d4aac94d3fd16809ee413bc99294a085798a589dae51ddd4a", "safeBlockHash": "0x3b8fb240d288781d4aac94d3fd16809ee413bc99294a085798a589dae51ddd4a", "finalizedBlockHash": "0x0000000000000000000000000000000000000000000000000000000000000000"},{"timestamp": "0x5","prevRandao": "0x0000000000000000000000000000000000000000000000000000000000000000","suggestedFeeRecipient": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"}],"id":67}' http://127.0.0.1:8550
+        ```
+
+    === "JSON result"
+
+        ```json
+        {
+            "jsonrpc": "2.0",
+            "id": 67,
+            "result": {
+              "payloadStatus": {
+                "status": "VALID",
+                "latestValidHash": "0x3b8fb240d288781d4aac94d3fd16809ee413bc99294a085798a589dae51ddd4a",
+                "validationError": null
+            },
+            "payloadId": "0x0000000021f32cc1"
+          }
+        }
+        ```
+
+### 2. Get the payload
+
+Get the payload with [`engine_getPayloadV1`](../../../Reference/Engine-API-Methods.md#engine_getpayloadv1)
```

Comment:
```suggestion
Get the payload using [`engine_getPayloadV1`](../../../Reference/Engine-API-Methods.md#engine_getpayloadv1).
```

Use consistent wording/syntax for parallel descriptions.

Reply:
Fixed